pyproject.toml

```diff
@@ -1,11 +1,14 @@
-[project]
+[tool.poetry]
 name = "vessim"
-dynamic = ["version", "description"]
-readme = "README.md"
-requires-python = ">=3.8"
-license = {file = "LICENSE"}
+version = "0.1.0"
+description = "co-simulation testbed for carbon-aware applications and systems"
```

Comment:
Is there a way to get the version from `vessim/__init__.py`? otherwise let's remove it there so we only have one source of truth